### PR TITLE
Use env var for nimble

### PR DESCRIPTION
**Pull request by genotrance**
Previous PR was just wrong - it finds the `nimble` binary for the active `nim e` which is still the broken `nimble`.

Instead, tester will now set the `NIMBLE_TEST` env var on startup which all child processes can use.

### CODE REVIEW

File: issue280and524.nimble [hunk 1]
```diff
@@ -10,15 +10,10 @@ srcDir        = "src"
...
-import strutils
-
 task setup, "Download and generate":
   let
-    dirSep = when defined(windows): "\\" else: "/"
-    path =
-      if selfExe().len == 0: ""
-      else: selfExe().rsplit(dirSep, maxsplit = 1)[0] & dirSep
-  exec path & "nimble install https://github.com/nimble-test/issue280and524.git?subdir=generator -y"
+    nimble = getEnv("NIMBLE_TEST", "nimble")
```
Comment (dom96) [inline issue280and524.nimble]: Call it ``NIMBLE_TEST_BINARY_PATH`` please.

**Review by dom96 — CHANGES_REQUESTED** [review]

File: issue280and524.nimble (hunk 1 above)
Comment (genotrance) [reply issue280and524.nimble]: Done

File: issue280and524.nimble [hunk 2]
```diff
@@ -10,15 +10,10 @@ srcDir        = "src"
...
-import strutils
-
 task setup, "Download and generate":
   let
-    dirSep = when defined(windows): "\\" else: "/"
-    path =
-      if selfExe().len == 0: ""
-      else: selfExe().rsplit(dirSep, maxsplit = 1)[0] & dirSep
-  exec path & "nimble install https://github.com/nimble-test/issue280and524.git?subdir=generator -y"
+    nimble = getEnv("NIMBLE_TEST_BINARY_PATH", "nimble")
```
Comment (dom96) [inline issue280and524.nimble]: i'd rather this fail then choose some random version of Nimble

**Review by dom96 — CHANGES_REQUESTED** [review]
sorry, another thing

File: issue280and524.nimble (hunk 2 above)
Comment (genotrance) [reply issue280and524.nimble]: Done

**Pull request merged**